src/components/space/charts/ActionsMap.tsx

```diff
@@ -52,9 +49,26 @@ const ActionsMap = (): JSX.Element | null => {
     actionsToChart = filterActionsByUsers(actions, selectedUsers);
   }
 
+  // NO Google key or no actions at all
+  if (!GOOGLE_KEY || !actionsToChart.length) {
+    return null;
+  }
+
   // GeoJSON Feature objects
   const points = mapActionsToGeoJsonFeatureObjects(actionsToChart);
 
+  // if there's actions but with no geolocation points
+  if (!points.length) {
+    return (
+      <>
+        <ChartTitle title={t('ACTIONS_BY_LOCATION')} />
```

Comment:
I think we want to show the alert above the map as well if `points.length !== actionsToChart.length`